src/project.js

```diff
@@ -0,0 +1,134 @@
+const newTodoInputDesc = document.querySelector('[data-new-todo-desc-input]');
+const newTodoInputPrior = document.querySelector('[data-new-todo-prior-input]');
+const newTodoInputDate = document.querySelector('[data-new-todo-date-input]');
+const newTodoInputTime = document.querySelector('[data-new-todo-time-input]');
+const newTodoInputNote = document.querySelector('[data-new-todo-note-input]');
+
+const addProject = document.getElementById('add_project');
+const addTask = document.getElementById('add_task');
+
+const deleteTask = document.getElementById('button_delete');
+const updateTask = document.getElementById('button_update');
+
+const LOCAL_STORAGE_ALL_PROJECTS = 'projects';
+const LOCAL_STORAGE_SELECTED_PROJECT = 'selectedProjectId1';
+const LOCAL_STORAGE_SELECTED_TASK = 'selectedTaskId1';
+
+const projects = JSON.parse(localStorage.getItem(LOCAL_STORAGE_ALL_PROJECTS)) || [
+  {
+    id: Date.now().toString(),
+    name: 'Default Project',
+    todos: [],
+  },
+];
+
+const renderdescription = (selectedProject) => {
+  descriptionContainer.innerHTML = '';
+  const selectedTaskId1 = localStorage.getItem(LOCAL_STORAGE_SELECTED_TASK);
+  const selectedTask = selectedProject.todos.find((task) => task.id === selectedTaskId1);
+  descriptionContainer.innerHTML = `Name:- ${selectedTask.name} <br> Description:-${selectedTask.desc}
+          <br> Priority:- ${selectedTask.prior} <br> Date:- ${selectedTask.date} <br> note:- ${selectedTask.note} <br>
+           `;
+  deleteTask.style.visibility = 'visible';
+  updateTask.style.visibility = 'visible';
+};
+
+const description = (task, selectedProject) => {
+  const selectedTaskId1 = task.id;
+  localStorage.setItem(LOCAL_STORAGE_SELECTED_TASK, selectedTaskId1);
+  renderdescription(selectedProject);
+  return selectedTaskId1;
+};
+
+const rendertask = () => {
+  taskContainer.innerHTML = '';
+  descriptionContainer.innerHTML = '';
+  deleteTask.style.visibility = 'hidden';
+  updateTask.style.visibility = 'hidden';
+  const selectedProjectId1 = localStorage.getItem(LOCAL_STORAGE_SELECTED_PROJECT);
+  const selectedProject = projects.find((project) => project.id === selectedProjectId1);
+  taskContainer.innerHTML = `Project Name:-${selectedProject.name}`;
+  selectedProject.todos.forEach((task) => {
+    const taskElement = document.createElement('li');
+    taskElement.dataset.taskId = task.id;
+    taskElement.innerText = task.name;
+    taskElement.onclick = () => { description(task, selectedProject); };
+    taskContainer.appendChild(taskElement);
+  });
+};
+
+const task = (project) => {
+  addTask.style.visibility = 'visible';
+  const selectedProjectId1 = project.id;
+  localStorage.setItem(LOCAL_STORAGE_SELECTED_PROJECT, selectedProjectId1);
+  rendertask();
+};
+
+const renderProjects = () => {
+  projectsContainer.innerHTML = '';
+  descriptionContainer.innerHTML = '';
+  deleteTask.style.visibility = 'hidden';
+  updateTask.style.visibility = 'hidden';
```

Comment:
- [ ] [REQUIRED] 
Keep all the styling logic in CSS files (If you need to change something like a background image, add the classes with the different backgrounds to the CSS and then change the class name attribute with JavaScript) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/section/styling_requirement.png) from Pathwright)